essence-of-live-coding/test/Cell.hs

```diff
@@ -23,10 +25,49 @@ import LiveCoding
 
 import qualified Cell.Util
 import qualified Cell.Monad.Trans
+import Control.Exception (ErrorCall)
+import Test.Framework.Providers.HUnit
+import Control.Exception.Base
+import Test.HUnit
 
 test = testGroup "Cell"
   [ testProperty "steps produces outputs"
     $ \(inputs :: [Int]) -> inputs === fst (runIdentity $ steps (id :: Cell Identity Int Int) inputs)
+  , testProperty "traverse' works as expected for Lists"
```

Comment:
What's the law you're expecting to hold?

Reply:
I was checking whether traverse' applies the function to each element of the list.

Reply:
So something like `traverse' (cell >>> arr f) = traverse' cell >>> arr (map f)`?